src/assets/app.js

```diff
@@ -450,6 +450,9 @@ app.controller('VersionsController', function($rootScope, $scope, sessionHttp, n
       if (!jiraBase && resp.data.jira_base) {
         jiraBase = resp.data.jira_base;
       }
+      if (resp.data.login_suffix) {
+        $scope.req.admin_user = loggedInUser() + resp.data.login_suffix;
```

Comment:
I think this resets it after submission. I think need to modify line 411 too.

Reply:
I think 411 (`init`) runs before the JS has made a request to the server, so I won't have the prefix at that point (unless `app.js` can read config.toml directly?). This spot works because it gets called for the dry run, so we've then got the prefix ready when we do the second "for real" run. Its the same pattern used for `jira.base_url`

Reply:
that is a good point. but the dry run is executed as the jira user, i thought. or does it use the jira service account? 

that makes sense then. :+1:

Reply:
I think that's correct -- the API impl uses `jira_config` as-is for dry run, and overwrites with the provided auth when its for real:
https://github.com/tanium/octobot/blob/9849a46c90febfba739d186059b5a92051f65a74/src/server/admin.rs#L244-L246